src/styles/global/print.scss

```diff
@@ -1,8 +1,96 @@
 //
 // CUSTOMIZED PRINT STYLES
+@import 'variables';
 
 @media print {
+  @page {
+    margin-top: 100px;
+  }
+
+  @supports (-moz-appearance: none) {
+    .logo {
```

Comment:
Please put the logo styling at one place (not at the top and the bottom of the file).